modules/sdk/gradle-plugins-workspace/src/gradleTest/themesNodeBuild/themes/baz-theme/.gitignore

```diff
@@ -0,0 +1,6 @@
+node_modules
+build
+dist
+liferay-theme.json
+.web_bundle_build
+.yo-rc.json
```

Comment:
Can we sort these lines so it matches the other .gitignore files in portal?

```
.web_bundle_build
.yo-rc.json
build
dist
liferay-theme.json
node_modules
```

Reply:
Fixed.